contracts/DCNT721A.sol

```diff
@@ -78,13 +103,19 @@ contract DCNT721A is ERC721A, Initializable, Ownable, Splits {
         _metadataConfig.metadataRendererInit
       );
     } else {
+      _contractURI = _metadataConfig.contractURI;
       baseURI = _metadataConfig.metadataURI;
     }
   }
 
-  function mint(uint256 numberOfTokens) external payable {
-    uint256 mintIndex = totalSupply();
-    require(saleIsActive, "Sale must be active to mint");
+  function mint(uint256 numberOfTokens)
+    external
+    payable
+    verifyTokenGate(false)
+  {
+    uint256 mintIndex = _nextTokenId();
+    require(block.timestamp >= saleStart, "Sales are not active yet.");
```

Comment:
>= / <= costs more gas than > < recommend updating for gas optimization.

https://nftchance.medium.com/the-gas-efficient-way-of-building-and-launching-an-erc721-nft-project-for-2022-b3b1dac5f2e1